app/build.gradle

```diff
@@ -18,8 +18,7 @@ android {
       testCoverageEnabled = false
     }
     release {
-      minifyEnabled false
-      proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
```

Comment:
Should we be removing the proguard file here?  Will our rules still be tested if we do?